packages/pkh-tezos/test/index.test.ts

```diff
@@ -1,5 +1,28 @@
-describe('Solana Cacao Auth Verify', () => {
-  test.skip('Verify', () => {
+import { InMemorySigner } from '@taquito/signer'
+import { verifySignature, getPkhfromPk } from '../src/verifier'
+import { char2Bytes, verifySignature as taquitoVerifySignature, getPkhfromPk as taquitoGetPkhfromPk } from '@taquito/utils'
```

Comment:
Can we just generate some test vectors so that we don't need to include the `@taquito` deps at all?